release/models/platform/openconfig-platform.yang

```diff
@@ -540,11 +546,13 @@ module openconfig-platform {
     }
 
     leaf model-name {
-      type string;
-      description
-        "A human readable string describing the model of a component.
-        This string is optional and should only be populated if part-no
-        is also populated.";
+      when "../removable = 'true'" {
```

Comment:
The type stmt must be defined outside of the when stmt - this conditional stmt has to be defined as a sibling of the leafs substatements (and can carry it's own separate description substatement)

Reply:
Oh I see, thanks!